src/main/java/org/broadinstitute/hellbender/tools/walkers/featuremapping/AddFlowSNVQuality.java

```diff
@@ -0,0 +1,601 @@
+package org.broadinstitute.hellbender.tools.walkers.featuremapping;
+
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.barclay.argparser.*;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.FlowBasedProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.GATKPath;
+import org.broadinstitute.hellbender.engine.ReadWalker;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.FlowBasedArgumentCollection;
+import org.broadinstitute.hellbender.tools.walkers.groundtruth.SeriesStats;
+import org.broadinstitute.hellbender.utils.read.FlowBasedRead;
+import org.broadinstitute.hellbender.utils.read.FlowBasedReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicReference;
+
+@CommandLineProgramProperties(
+        summary = "SNV Quality Tool (flow space processing)",
```

Comment:
```suggestion
        summary = "This program converts the flow qualities that Ultima Genomics CRAM reports to more conventional base qualities. Specifically, the reported BQ will report the probability that a specific base is a sequencing error mismatch, while auxilary tags qa, qt, qg, qc report specific probability that a specific base X is a A->X error. Since mismatch error in flow-based chemistries can only occur as a result of several indel errors, we implemented three strategies to estimate the probability of a mismatch:         
<ul>
        <li> Optimistic: mismatches occur as independent indel errors </li>
        <li> Pessimistic: indel errors that generate a mismatch are fully dependent </li> 
        <li> Geometric: some dependency between indel errors that generate mismatches </li>
</ul>        
 "
```